scripts/certificates.sh

```diff
@@ -14,6 +14,7 @@ init() {
     fi
     client bpfd
     client bpfctl
+    client go-bpfd-client
```

Comment:
Maybe also in the docs you can call out how to use `./scripts/certificates.sh client gocounter` to generate this.
Or to say that a user needs to generate their own cerficate for their app, signed by the same CA as the bpfd certificate.